bitnami/ghost/README.md

```diff
@@ -50,88 +50,98 @@ The command removes all the Kubernetes components associated with the chart and
+| `ingress.hosts[0].tlsSecret`         | TLS Secret (certificates)                                                                          | `ghost.local-tls-secret`                                                                      |
+| `ingress.secrets[0].name`            | TLS Secret Name                                                                                    | `nil`                                                                                         |
+| `ingress.secrets[0].certificate`     | TLS Secret Certificate                                                                             | `nil`                                                                                         |
+| `ingress.secrets[0].key`             | TLS Secret Key                                                                                     | `nil`                                                                                         |
+| `externalDatabase.host`              | Host of the external database                                                                      | `localhost`                                                                                   |
+| `externalDatabase.port`              | Port of the external database                                                                      | `3306`                                                                                        |
+| `externalDatabase.user`              | Existing username in the external db                                                               | `bn_ghost`                                                                                    |
+| `externalDatabase.password`          | Password for the above username                                                                    | `""`                                                                                          |
+| `externalDatabase.database`          | Name of the existing database                                                                      | `bitnami_ghost`                                                                               |
+| `mariadb.enabled`                    | Whether or not to install MariaDB (disable if using external)                                      | `true`                                                                                        |
+| `mariadb.rootUser.password`          | MariaDB admin password                                                                             | `nil`                                                                                         |
+| `mariadb.db.name`                    | MariaDB Database name to create                                                                    | `bitnami_ghost`                                                                               |
```

Comment:
The following values are set in the values.yaml file but not explained here:

- mariadb.replication.enabled

- mariadb.master.persistence.enabled

- mariadb.master.persistence.accessMode

- mariadb.master.persistence.size

Reply:
done